website/docs/r/sentinel_alert_rule_fusion.html.markdown

```diff
@@ -50,6 +41,7 @@ resource "azurerm_sentinel_alert_rule_fusion" "example" {
 The following arguments are supported:
 
 * `name` - (Required) The name which should be used for this Sentinel Fusion Alert Rule. Changing this forces a new Sentinel Fusion Alert Rule to be created.
```

Comment:
Can name ONLY be `BuiltInFusion`? if so could we please add that as a default, make the property optional, and deprecate it.

or can additional rules be added? or the existing ones name renamed?

Reply:
I'm not sure about the already existing rules with different names, for new created ones, it can only be `BuiltInFusion`